src/App.vue

```diff
@@ -60,34 +65,63 @@
         if (validateUri(this.url)) {
           this.request(this.url);
         } else {
-          this.setErrorMessage(invalidUrlErrorMessage);
+          this.handleNoResponse(invalidUrlErrorMessage);
         }
       },
-      request(url) {
+      request(url, method) {
         const vm = this;
 
-        fetch(url, { headers: { Accept: "application/json" } })
+        fetch(url, {
+          headers: { Accept: "application/json" },
+          method: method
+        })
           .then((response) => {
-            vm.handleResponse(response);
+            const statusCode = response.status;
+            const allowHeaderValue = response.headers.get("Allow");
```

Comment:
you should probably put this after line 84 since you don't need it unless it goes to line 84 or I'm also okay with this not being a variable at all, seems self explanatory

Reply:
I wanted to assign it to a variable first, because i will still have it used 2 times in the code (in the `if` expression, and in the `this.setAllowDelete` statement